projects/rocm-core/rdhc/rdhc.py

```diff
@@ -1748,7 +1748,8 @@ def main():
     parser = argparse.ArgumentParser(description="ROCm Deployment Health Check Tool",
                                     formatter_class=argparse.RawDescriptionHelpFormatter,
                                     usage="sudo -E ./rdhc.py [options]",
-                                    epilog="Usage examples:\n"+
+                                    epilog="Refer the README @<ROCM_INSTALL_PATH>/share/rdhc/README.md \n"+
```

Comment:
Can we rename this README.md to have unique name or integrate with Main module level README?
install folder to consider share/doc/rocm-core/?